src/LibraryManager.Contracts/Resources/Text.resx

```diff
@@ -178,4 +178,7 @@
     <value>"{0}" does not contain the following: {1}
 Valid files are {2}</value>
   </data>
-</root>
+  <data name="ErrorDuplicateLibraries" xml:space="preserve">
```

Comment:
This error message should be: "Cannot restore. Multiple definitions for library: {LibraryName}"

We could have multiple duplicates. Suggestions one the text?